[Bug]: Breaked bash path in Makefile

### Prerequisites

- [x] I have searched existing issues and discussions to avoid duplicates
- [x] I am using the latest version (or have tested against main/nightly)

### Description

On nonFHS systems bash is not in the standard path

### Steps to reproduce

```
make <some-command>
```

### Expected behavior

just work

### Actual behavior

error

### Affected area(s)

Docs, Core (Go)

### Version

v1.5.0

### Environment

```text
OS: NixOS
```

### Relevant logs/output

```shell
make: /usr/bash/env: Нет такого файла или каталога make: *** [Makefile:69: cleanup-enterprise] Ошибка 127
```

### Regression?

_No response_

### Severity

Low (minor issue or cosmetic)

--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ NC=\033[0m # No Color
 ECHO := printf '%b\n'
 
 # nvm requires bash-compatible shell semantics; /bin/sh is dash on some Linux distros.
-SHELL := /bin/bash
+SHELL := /usr/bin/env bash
 
 # Ensures the Node version pinned in .nvmrc is active before any npm/node call.
 # nvm is a shell function, so each recipe that needs it must inline this snippet

